### PR TITLE
Add ctest for some test targets

### PR DESCRIPTION
**Pull request by ilyachur**
### Details:
 - Add ctest for some test targets

### Tickets:
 - NA

### CODE REVIEW

File: docs/dev/test_coverage.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ And build OpenVINO as usual.
 In order to generate coverage reports, first of all, the tests must be run. Depending on how many tests are run, the better covegare percentage can be achieved. E.g. for `openvino` component, `InferenceEngineUnitTests`, `ieUnitTests`, `ieFuncTests` must be run as well as plugin tests.
 
 ```bash
-$ ctest -V -L IE
+$ ctest -V
```
Comment (ilya-lavrenov) [inline docs/dev/test_coverage.md:37]: do we have a replacement for IE label?

File: src/frontends/onnx/tests/CMakeLists.txt [hunk 2]
```diff
@@ -122,6 +122,7 @@ foreach(BACKEND_NAME ${ACTIVE_BACKEND_LIST})
 endforeach()
 
 add_executable(ov_onnx_frontend_tests ${SRC})
+add_test(NAME ov_onnx_frontend_tests COMMAND ov_onnx_frontend_tests)
```
Comment (ilya-lavrenov) [inline src/frontends/onnx/tests/CMakeLists.txt:125]: I suggest adding labels.
Now, we have CPU, GPU, IE, may be others.

File: docs/dev/test_coverage.md (hunk 1 above)
Comment (ilyachur) [reply docs/dev/test_coverage.md:37]: Just run all tests

File: src/frontends/onnx/tests/CMakeLists.txt (hunk 2 above)
Comment (ilyachur) [reply src/frontends/onnx/tests/CMakeLists.txt:125]: Fixed

**Review by ilya-lavrenov — APPROVED** [review]

**Pull request merged**